src/app.js

```diff
@@ -181,11 +171,20 @@ app.on('ready', () => {
   if (process.platform === 'darwin') {
     app.dock.hide();
   }
+
+  if (store.get('username') === '') {
+    tray.setImage(icon.fail);
+    createUsernameWindow();
+    tray.setContextMenu(
+      createTrayMenu('0', '0', '0')
+    );
+    return;
+  }
   
   app.on('window-all-closed', () => {});
   tray.on('right-click', requestContributionData);
   ipcMain.on('setUsername', setUsername);
 
   requestContributionData();
-  setInterval(requestContributionData, 1000 * 60 * 15); // 15 Minutes
+  setInterval(requestContributionData, 10000); // 15 Minutes
```

Comment:
Why was this changed?

Reply:
I undo it in the next commit, because of debug purpose

Reply:
Ah, sorry I didn't see that you changed it back!